src/test/java/io/cdap/plugin/gcp/bigquery/relational/BigQueryRelationTest.java

```diff
@@ -577,6 +655,169 @@ public void testCollectDeduplicateAggregationDefinitionErrors() {
     filterFields.clear();
   }
 
+  @Test
+  public void testSupportsWindowAggregationDefinition() {
```

Comment:
From what I can see there are multiple test scenarios being tested in a single test here. Can we divide these into different methods so that we can see which test breaks at a glance?

Reply:
Followed the pattern of existing unit tests for other pushdowns. We are asserting for every field.So it is easy to identify at what place the test breaks.